_includes/_boinc_1A.htm

```diff
@@ -29,13 +24,12 @@ <h4 class="text-bold">Install BOINC</h4>
         <!--Step 1-->
         <div class="row">
             <div class="col-6 col-sm-6 col-md-6 col-lg-6">
-                <p>Navigate to the <a href="https://boinc.berkeley.edu">BOINC website</a>, download and install the appropriate BOINC client on each of the computers you wish to crunch BOINC work upon. If you're not crunching projects which require <a href="https://www.virtualbox.org/wiki/Downloads">VirtualBox</a> you don't need to install BOINC+VirtualBox.</p>
+                <p>Navigate to the <a href="https://boinc.berkeley.edu">BOINC website</a>, download and install the appropriate BOINC client on each of the computers you wish to crunch BOINC work upon. To be able to crunch the widest variety of BOINC projects, we suggest the BOINC + Virtualbox bundle.</p>
```

Comment:
Is this a valid statement? What whitelisted projects require VBox?

Reply:
I don't know about _require_ but certainly a project in the future might. Rosetta has vbox tasks, idk about others. Basically:

- There is no reason _not_ to install Virtualbox. The only scenario I can imagine is if a user already has vbox installed and needs it kept at a particular version in which case they will know how to navigate this anyways.
- Some projects will want Virtualbox, and users will complain if they somehow can't crunch a project or certain WU types from that project

Reply:
> There is no reason not to install Virtualbox

If a user already has VM or hypervisor installed on their machine, there is a very good reason not to install VBox

Reply:
Right, so such a user who knows wtf a hypervisor is will have no problem figuring out they don't need to install it with the bundled version. 95% of users do not have any hypervisor installed on their system. I don't know the actual % of course, but I know it is a minority of users and of those users, we expect them to be rather tech savvy.

Reply:
Also this does not imply that it's required, only that some projects may want it, and it doesn't require our 95% user to figure out what their project requires which may or may not be listed on the project website in an easy to find place.